modules/system/chat-wfrp4e.js

```diff
@@ -443,12 +452,12 @@ export default class ChatWFRP {
       return ui.notifications.error(game.i18n.localize("CONDITION.ApplyError"))
 
     if (game.user.isGM)
-      message.update(conditionResult)
+      await message.update(conditionResult)
     else
-      game.socket.emit("system.wfrp4e", { type: "updateMsg", payload: { id: msgId, updateData: conditionResult } })
+      await WFRP_Utility.awaitSocket(game.user, "updateMsg", { id: msgId, updateData: conditionResult }, "executing condition script");
   }
 
-  static _onApplyEffectClick(event) {
+  static async _onApplyEffectClick(event) {
```

Comment:
Don't see the point of async here (populateEffect isn't async)

Reply:
removed the await on populate effect.